src/env/clean_up.rs

```diff
@@ -24,7 +24,12 @@ pub fn register_signals() -> anyhow::Result<Arc<AtomicBool>> {
     let terminate = Arc::new(AtomicBool::new(false));
     signal_hook::flag::register(signal_hook::SIGTERM, Arc::clone(&terminate))?;
     signal_hook::flag::register(signal_hook::SIGINT, Arc::clone(&terminate))?;
-    signal_hook::flag::register(signal_hook::SIGQUIT, Arc::clone(&terminate))?;
+
+    #[cfg(not(windows))]
```

Comment:
:question: Will clean-up work the same on Windows then?

Reply:
Yes should be fine as `CTRL+C` is `SIGINT` on windows: https://docs.microsoft.com/en-us/windows/console/ctrl-c-and-ctrl-break-signals